packages/cms-cli/commands/init.js

```diff
@@ -34,83 +34,101 @@ const {
+    logger.error(`The config file '${configPath}' already exists.`);
+    logger.info(
+      'To update an existing config file, use the "hs auth" command.'
+    );
+    process.exit(1);
+  }
 
-      if (configPath) {
-        logger.error(`The config file '${configPath}' already exists.`);
-        logger.info(
-          'To update an existing config file, use the "hs auth" command.'
-        );
-        process.exit(1);
-      }
+  trackAuthAction(
+    COMMAND_NAME,
+    PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
+    TRACKING_STATUS.STARTED
+  );
 
-      trackAuthAction(
-        COMMAND_NAME,
-        PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-        TRACKING_STATUS.STARTED
-      );
+  createEmptyConfigFile();
+  handleExit(deleteEmptyConfigFile);
 
-      createEmptyConfigFile();
-      handleExit(deleteEmptyConfigFile);
+  try {
+    const configData = await personalAccessKeyPrompt({ env });
+    const { name } = await promptUser([PORTAL_NAME]);
 
-      try {
-        const configData = await personalAccessKeyPrompt({ env });
-        const { name } = await promptUser([PORTAL_NAME]);
+    await updateConfigWithPersonalAccessKey(
+      {
+        ...configData,
+        name,
+      },
+      true
+    );
 
-        await updateConfigWithPersonalAccessKey(
-          {
-            ...configData,
-            name,
-          },
-          true
-        );
+    const path = getConfigPath();
+    const portalId = getPortalId();
 
-        const path = getConfigPath();
-        const portalId = getPortalId();
+    logger.success(
+      `The config file "${path}" was created using your personal access key for portal ${portalId}.`
+    );
 
-        logger.success(
-          `The config file "${path}" was created using your personal access key for portal ${portalId}.`
-        );
+    trackAuthAction(
+      COMMAND_NAME,
+      PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
+      TRACKING_STATUS.COMPLETE,
+      portalId
+    );
+  } catch (err) {
+    logErrorInstance(err);
+    trackAuthAction(
+      COMMAND_NAME,
+      PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
+      TRACKING_STATUS.ERROR
+    );
+  }
+};
 
-        trackAuthAction(
-          COMMAND_NAME,
-          PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-          TRACKING_STATUS.COMPLETE,
-          portalId
-        );
-      } catch (err) {
-        logErrorInstance(err);
-        trackAuthAction(
-          COMMAND_NAME,
-          PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-          TRACKING_STATUS.ERROR
-        );
-      }
-    });
+// Yargs Configuration
+const command = `${COMMAND_NAME}`;
+const describe = DESCRIPTION;
+const builder = yargs => {
+  addLoggerOptions(yargs, true);
+  addTestingOptions(yargs, true);
+
+  return yargs;
+};
+const handler = async argv => action(argv);
+
+// Commander Configuration
+const configureCommanderInitCommand = program => {
+  program
+    .version(version)
+    .description(DESCRIPTION)
+    .action(async (command = {}) => action(command));
```

Comment:
What is the reason for the extra `() => {}`? Could the initialization of the param happen in the declaration of `action`?

Reply:
No reason other than I was just following convention of the other modules.  Sure, I can do the initialization over there